### DISCUSSION TITLE
Data compression

### DISCUSSION THREAD

**Discussion by aydar-s-v37** [datalust/seq-tickets#2207] [Help] [answered]
Hi there! I'm trying to understand how storage data compression works. Here is storage use graph:

![storage](https://github.com/datalust/seq-tickets/assets/139084479/3595afd2-c2c7-4a40-ae09-7573403910be)

Older data is ingested from different seq instance using clef tool. Recent data (from April onvards) is ingested directly to seq instance, and obviously takes much more space. Amount of evens is within same range, so I'm guessing imported/older data is heavily compressed. How compression works? Is there a way for force compression for anything older than X days to save disk space?

**Comment by liammclennan** [comment, marked as answer]
Hi @aydar-viv 

The compression is the same across the entire stream. You can use the histogram on the events page to see how event volume has changed over time, or the 'Space by event type' built-in query to check the data size by event type. If the event volume is the same then it must be that the newer events carry more data. The two tools you have available to manage disk usage are:

* set a filter on the API key to reduce ingestion
* use retention policies to delete events when they are no longer needed

**Reply by aydar-s-v37** [reply]
Interesting... Asking because I noticed this:
* data on server A was taking let's say 150MB per day
* I exported data to server B using clef tool and it is taking 15MB per day

**Reply by liammclennan** [reply]
Next step is probably to export the data from both and do a text diff.

**Reply by aydar-s-v37** [reply]
Yeah I think I messed up with initial data import from different server and it was only partially transferred. Thanks!

**Reply by liammclennan** [reply]
Glad you found a solution.
